docs/e2e/e2e-checks.md

```diff
@@ -21,36 +21,59 @@ In order to support e2e for multiple apps, the folder structure will include a b
 
 Some highlights:
 - By default, the base config is defined to run on a minimal browser-set (desktop and mobile chrome). Browsers can be added in the app-specific playwright config.
-- Snapshots will be output locally or in the artifacts of the CI job
+- Snapshots will be output locally (in the `./e2e` folder or the container) - or in the artifacts of the CI job
 - HTML reports are output to the `playwright-report` folder
-- Parallelism limited on CI to ensure stable execution
 - Accessibility testing can be performed using the `@axe-core/playwright` package (https://playwright.dev/docs/accessibility-testing)
 
+## Running Tests Locally
 
-## Running Locally
+### Running Tests with Docker (preferred)
```

Comment:
Style nits:
- Use imperative verbs for instructions "Run tests" rather than "Running tests"
- Use sentence case "Run tests" rather than "Run Tests"